mesheryctl/internal/cli/root/root.go

```diff
@@ -19,7 +19,7 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/app"
```

Comment:
Good deal. Removing reference in `root` 👍

Reply:
Thanks.